lib/cylc/gui/scanutil.py

```diff
@@ -250,7 +250,119 @@ def get_scan_menu(suite_keys,
+        gcylc_item.set_image(img_gcylc)
+        gcylc_item._connect_args = (host, owner, suite)
+        gcylc_item.connect(
+            "button-press-event",
+            lambda b, e: launch_gcylc(b._connect_args))
+        gcylc_item.show()
+        menu.append(gcylc_item)
+
+    sep_item = gtk.SeparatorMenuItem()
+    sep_item.show()
+    menu.append(sep_item)
+
+    # Construct a cylc stop item to stop a suite
+    if len(suite_keys) > 1:
+        stoptask_item = gtk.ImageMenuItem('Stop all...')
+    else:
+        stoptask_item = gtk.ImageMenuItem('Stop...')
+
+    stop_menu = gtk.Menu()
+    stoptask_item.set_submenu(stop_menu)
+    img_stop = gtk.image_new_from_stock(gtk.STOCK_MEDIA_STOP,
+                                        gtk.ICON_SIZE_MENU)
+    stoptask_item.set_image(img_stop)
+
+    for stop_type in ['', '--kill', '--now', '--now --now']:
+        item = gtk.ImageMenuItem('stop %s' % stop_type)
+        img_stop = gtk.image_new_from_stock(gtk.STOCK_MEDIA_STOP,
+                                            gtk.ICON_SIZE_MENU)
+        item.set_image(img_stop)
+        stop_menu.append(item)
+        item._connect_args = suite_keys, 'stop %s' % stop_type
+        item.connect(
+            "button-press-event",
+            lambda b, e: call_cylc_command(b._connect_args[0],
+                                           b._connect_args[1]))
+        item.show()
+
+    stoptask_item.show()
+    menu.append(stoptask_item)
+
+    # Construct a cylc hold item to hold (pause) a suite
+    if len(suite_keys) > 1:
+        holdtask_item = gtk.ImageMenuItem('Hold all')
+    else:
+        holdtask_item = gtk.ImageMenuItem('Hold')
+
+    img_hold = gtk.image_new_from_stock(gtk.STOCK_MEDIA_PAUSE,
+                                        gtk.ICON_SIZE_MENU)
+    holdtask_item.set_image(img_hold)
+    holdtask_item._connect_args = suite_keys, 'hold'
+    holdtask_item.connect("button-press-event",
+                          lambda b, e: call_cylc_command(b._connect_args[0],
+                                                         b._connect_args[1]))
+    menu.append(holdtask_item)
+    holdtask_item.show()
+
+    # Construct a cylc release item to release a paused/stopped suite
+    if len(suite_keys) > 1:
+        unstoptask_item = gtk.ImageMenuItem('Release all')
+    else:
+        unstoptask_item = gtk.ImageMenuItem('Release')
+
+    img_release = gtk.image_new_from_stock(gtk.STOCK_MEDIA_PLAY,
+                                           gtk.ICON_SIZE_MENU)
+    unstoptask_item.set_image(img_release)
+    unstoptask_item._connect_args = suite_keys, 'release'
+    unstoptask_item.connect("button-press-event",
+                            lambda b, e: call_cylc_command(b._connect_args[0],
+                                                           b._connect_args[1]))
+    unstoptask_item.show()
+    menu.append(unstoptask_item)
+    _add_main_menu_item(menu)
+
+    return menu
+
+
+def launch_about_dialog(program_name, hosts):
```

Comment:
`get_gpanel_scan_menu` (line 233 and line 247) above still ccalling `_launch_about_dialog` and `_launch_hosts_dialog`.